hacktobermap.js

```diff
@@ -1,4 +1,10 @@
 var hacktobermap = [
+  {
+    name: 'Ryan Sperzel',
+    location: 'New Haven, CT',
+    latitude: 41.308273,
+    longitude: -72.927879,
+  }
```

Comment:
Thanks for contributing! Looks like you forgot to add a comma after the closing brace.